aboutusv1.html

```diff
@@ -0,0 +1,56 @@
+<!doctype html>
+<html> 
+<head>
+    <title>JumboCode: About</title>
+</head>
+
+<body>
+
+    <div>
+        <h2>Who We Are</h2>
+        <p>blah blah blah</p>
```

Comment:
Though this is just a skeleton structure, just leave these lines out: `<p>blah blah blah</p>`. It makes the code look cleaner.